nservicebus/messaging/databus/index.md

```diff
@@ -19,9 +19,23 @@ Instead of serializing the payload along with the rest of the message, the `Data
 
 If the location is not available upon sending, the send operation will fail. When a message is received and the payload location is not available, the receive operation will fail as well, resulting in the standard NServiceBus retry behavior, possibly resulting in the message being moved to the error queue if the error could not be resolved.
 
-## Alternative
+## Transport message size limits
 
-The [Handling large stream properties via pipeline](/samples/pipeline/stream-properties/) sample demonstrates a purely stream-based approach (rather than loading the full payload into memory) implemented by leveraging the NServiceBus pipeline.
+Using the Data Bus is required when the message size can exceed the transport message size limit.
+
+Note: Not all transports have very restrictive message size limits and Azure Service Bus has increased its size limits over the years. Check the respective transport website documentation for the latest maximum limit of the message size.
+
+| Transport                  | Maximum size |
+| -------------------------- | ------------:|
+| Amazon SQS                 | 256KB        |
+| Amazon SQS + S3            | 2GB          |
+| Azure Storage Queues       | 64KB         |
+| Azure Service Bus Standard | 256KB        |
+| Azure Service Bus Premium  | 100MB        |
+| RabbitMQ                   | No limit     |
+| SQL Server                 | No limit     |
+| Learning                   | No limit     |
+| MSMQ                       | 4MB          |
```

Comment:
The note says to "check the respective transport website documentation for the latest maximum limit of the message size", but we don't link to them in the table below, where it would be most convenient for the user to find them.

Reply:
@soujay was this comment deliberately ignored or was it missed?

Reply:
ping

Reply:
We have created a [PR](https://github.com/Particular/docs.particular.net/pull/6238) to link to message sizes - @soujay could you please review?